src/main/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/service/VisitService.kt

```diff
@@ -351,14 +328,14 @@ class VisitService(
 
     val visit = VisitDto(visitRepository.saveAndFlush(visitToBook))
 
-    // TODO can we change the name of the track event name?
-    telemetryClient.trackEvent(
-      "visit-scheduler-prison-visit-updated",
+    trackEvent(
+      TelemetryVisitEvents.VISIT_BOOKED_EVENT.eventName,
       listOfNotNull(
         "reference" to visitToBook.reference,
         "visitStatus" to visit.visitStatus.name,
-      ).toMap(),
-      null
+        "applicationReference" to visit.applicationReference,
+        "isUpdated" to changedVisit.toString()
+      ).toMap()
     )
 
     if (changedVisit) {
```

Comment:
Could possible have an event to say a booked visit has changed... but could add that later when we need it rather than if

Reply:
Yes that was the plan earlier. Changed it based on the Friday call. With this approach the event will closely match the API call.